keps/sig-storage/596-csi-inline-volumes/README.md

```diff
@@ -74,18 +126,24 @@ spec:
               foo: bar
 ```
 
-## Ephemeral inline volume proposal
-A CSI driver may be able to support either PV/PVC-originated or pod spec originated volumes. When a volume definition is embedded inside a pod spec, it is considered to be an `ephemeral inline` volume request and can only participate in *mount/unmount* volume operation calls.  Ephemeral inline volume requests have the following characteristics: 
-* The inline volume spec will not contain nor require a `volumeHandle`.
-* The CSI Kubelet plugin will internally generate a `volumeHandle` which is passed to the driver.
-* Using existing strategy, the volumeHandle will be cached for future volume operations (i.e. unmount).
-* The Kubelet will send mount related calls to CSI drivers:
-  * Kubelet will have access to both podUID and pod namespace during mount/Setup operations.
-  * Secrets references can be fully realized during mount/Setup phase and sent to driver.
-* The Kubelet will send unmount related calls to CSI drivers:
-  * The cached volumeHandle will be sent to the driver during unmount/Teardown phase.
+### Risks and Mitigations
+
+#### Security Considerations
+
+CSI driver vendors that support inline volumes will be responsible for secure handling of volumes.
+
+For example, `csi-driver-nfs` allows anybody who can create a pod to mount any NFS volume into that pod, when the cluster admin deploys the driver with csi driver instance allowing ephemeral use. This option is not on by default, but may be surprising to some admins that this allows mounting of any NFS volume and could be unsafe without additional authorization checks.
+
+Downstream distributions and cluster admins that wish to exercise fine-grained control over which CSI drivers are allowed to use ephemeral inline volumes within a pod spec should do so with a 3rd party pod admission plugin or webhook (not part of this KEP).
+
+We will update the documentation to include the security aspects of inline CSI volumes and recommend CSI driver vendors not implement inline volumes for persistent storage unless they also provide a 3rd party pod admission plugin.
```

Comment:
I think it's worth making sure that this documentation lands _before_ graduation, to give fair warning to anyone who upgrades and hasn't previously been told that their existing configuration may not be secure.

Reply:
Updated the graduation criteria with a reference to this section and one other section that requires doc updates prior to graduation.